index.html

```diff
@@ -1389,6 +1381,82 @@
       <a>Web NFC message</a> can be set for sending.
     </p>
   </section>
+  <section> <h3>The <strong>NFCWatchOptions</strong> dictionary</h3>
+      <p>
+        To describe which messages an application is interested in, the
+        <dfn>NFCWatchOptions</dfn> dictionary is used:
+      </p>
+      <pre class="idl">
+        enum NFCWatchMode {
+          "web-nfc-only",
+          "any"
+        };
+
+        dictionary NFCWatchOptions {
+          USVString url = null;
+          USVString kind = null;
+          USVString type = null;
+          NFCWatchMode mode = "web-nfc-only";
+        };
+      </pre>
+      <p>
+        The <dfn>NFCWatchOptions.url</dfn> property
+        denotes the <a>URL pattern</a> which is used for matching the
+        URL <a>origin</a> and <a>URL path</a> of the <a>Web NFC message</a>
+        which is stored in the <a>Web NFC record</a>.
+        The values <code>null</code>, <code>undefined</code>, and
```

Comment:
On the other hand, `""` as a default could make sense. Then you'd avoid having 3 representations of the same thing. https://heycam.github.io/webidl/#es-to-dictionary would convert both absent fields and explicit `undefined`s to `""`.